druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -755,28 +779,16 @@ private void checkGroupBySingleSortLimit(boolean approx) {
   /** Tests a query that contains no GROUP BY and is therefore executed as a
    * Druid "select" query. */
   @Test public void testFilterSortDesc() {
-    final String sql = "select * from \"foodmart\"\n"
```

Comment:
`select *` is expanded to `select col_1, col_2, ..., col_n` in the validation phase, so by the time the code that checks for roll ups sees it, it will throw an error because we're not supposed to select a virtual column. I thought about skipping cases with `select *` in them, but then queries like `select virtualcol from (select * from table)` would be allowed when they shouldn't be.